sympy/concrete/summations.py

```diff
@@ -429,6 +435,15 @@ def is_convergent(self):
 
         order = O(sequence_term, (sym, S.Infinity))
 
+        ### ----------- ratio test ---------------- ###
+        next_sequence_term = sequence_term.xreplace({sym: sym + 1})
+        ratio = combsimp(powsimp(next_sequence_term/sequence_term))
+        lim_ratio = limit(ratio, sym, upper_limit)
+        if abs((lim_ratio)) > 1:
```

Comment:
why double parentheses here, and in the < 1 comparison below?

Reply:
Sorry, it's a typo..